packages/odyssey-react/src/components/Icon/SvgIcon.tsx

```diff
@@ -63,4 +64,11 @@ const SvgIcon = forwardRef<SVGSVGElement, Props>(
   }
 );
 
+SvgIcon.displayName = "SvgIcon";
+
+SvgIcon = withStyles(styles)(SvgIcon);
```

Comment:
now that we're wrapping this component in our HOC, we also need to instruct our build tooling for storybook docs on how to behave in development and production environments. Should be as simple as adding `Icon` to the list here: https://github.com/okta/odyssey/blob/develop/packages/odyssey-storybook/src/index.ts#L21-L29